### PR TITLE
Added the `container-vm` OS

### PR DESCRIPTION
**Pull request by gabipetrovay**

### CODE REVIEW

**Comment by stephenplusplus** [comment]
Thanks for this! Can you strip this down to just the changes to index.js?

File: package.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
   "author": "Stephen Sawchuk <sawchuk@gmail.com>",
   "license": "MIT",
   "devDependencies": {
-    "mocha": "^2.2.5"
+    "mocha": "^2.2.5",
+    "describe": "^1.2.0"
```
Comment (stephenplusplus) [inline package.json]: describe is a global included with mocha, not an extra dependency :)

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+node_modules/
+.DS_Store
+*.swp
+npm-debug.log
```
Comment (stephenplusplus) [inline .gitignore]: I'm trying to phase these out from my repos, instead encouraging global gitignore files for this stuff.

**Comment by gabipetrovay** [comment]
@stephenplusplus You can delete your comment above. :smile:

**Comment by stephenplusplus** [comment]
Woo, thanks! Would you mind updating the readme, too?

**Comment by gabipetrovay** [comment]
There you go!

**Comment by stephenplusplus** [comment]
Wee-hee! Thanks again. Will release momentarily :)

**Comment by stephenplusplus** [comment]
Released as `0.2.2`

**Pull request merged**